content/id/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,107 @@
+---
+title: Menggunakan Minikube Untuk Membuat Klaster
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+      <div class="col-md-8">
+          <h3>Objectives</h3>
+                <ul>
+                    <li>Belajar apa itu klaster Kubernetes.</li>
+                    <li>Belajar apa itu Minikube.</li>
+                    <li>Memulai klaster Kubernetes menggunakan terminal _online_.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>Klaster Kubernetes</h3>
+                <p>
+                <b>Kubernetes mengoordinasikan klaster komputer ketersediaan tinggi (_highly available_) yang saling terhubung sebagai unit tunggal.</b> Abstraksi pada Kubernetes mengizinkan kamu untuk men-_deploy_ aplikasi terkemas (_containerized_) ke sebuah klaster tanpa perlu membalutnya secara spesifik pada setiap mesin. Untuk menggunakan model baru _deployment_ ini, aplikasi perlu dikemas dengan cara memisahkan mereka dari hos individu: mereka perlu dikemas. Aplikasi terkemas lebih fleksibel dan tersedia dibanding model _deployment_ lama, dimana aplikasi dipasang secara langsung didalam mesin spesifik sebagai paket yang sangat terintegrasi dengan hos. <b>Kubernetes mengotomasisasikan distribusi dan penjadwalan kontainer aplikasi sebuah klaster secara menyeluruh dengan cara yang lebih efisien.</b> Kubernetes merupakan platform _open-source_ dan siap produksi.
+                </p>
+                <p>Klaster Kubernetes terdiri dari 2 tipe sumber daya:
+                    <ul>
+                        <li><b>Master</b> mengoordinasikan klaster</li>
+                        <li><b>Node</b> adalah pekerja (_worker_) yang menjalankan aplikasi</li>
```

Comment:
There are some words that have underscores here. I think they need to be fixed since it is .html not .md.